core/storage/base_model/gae_models.py

```diff
@@ -1821,13 +1826,14 @@ def get_export_policy(cls) -> Dict[str, EXPORT_POLICY]:
             'content': EXPORT_POLICY.NOT_APPLICABLE
         })
 
-    # TODO(#13523): Change 'content' to domain object/TypedDict to
-    # remove Any from type-annotation below.
+    # Here we use type Any because the method 'create' defined in subclasses
+    # of BaseSnapshotContentModel can accept different Dict/TypedDict types.
+    # So, to allow every Dict/TypedDict type we used any here.
```

Comment:
Er, no, same concern as above.

Reply:
Since this method is defined in base class and redefined in other subclasses -- the above reasoning also applied to here